portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I love K-pop', 'I  hate snakes', 'Kdrama has been my go-to in this quarantine', 'people I know in Google are nice!','I don\'t know much about front-end, mostly because I imagine that I have to design stuffs', 'I dont like working from home', 'Hate to miss the Google office experience :(' ];
```

Comment:
Would be nice to break this list into multiple lines to avoid having a very long one. It will be easier to read this way :)